### ISSUE TITLE
Bug BBQ:  Plotting lesson conversion testing 

### ISSUE THREAD

**Issue by wrightaprilm** [datacarpentry/python-ecology-lesson#251]
@stijnvanhoey has graciously contributed a port of the current plotting lesson to PlotNine. If people would be willing to run through the lesson, it would be much appreciated. 

Lesson is discussed and linked in #248 .
More discussion of the plotting lesson can be found in issue #246 

Testing checklist: 

- [x]   Mac

- [x] Windows

- [x] Linux

**Comment by guadabsb15** [comment]
I did a pass through the lesson in #248  using Linux(Fedora), python3.6 and plotnine==0.3.0 and as data I used the suveys.csv file from lesson #2 (Starting with Data). Here are a few comments:

1. In the faceting section, for the last example p9.facet_grid("year ~ sex") would not work giving the error
    IndexError: index 0 is out of bounds for axis 0 with size 0 
    Which I am not sure it can be attributed to the fact that I am using a different data file, but commenting out that line will produce a plot 

2. Similarly with the solution for the second solution of the facetting challenge,  p9.facet_wrap("sex") would give PlotnineError: 'At least one layer must contain all variables used for facetting'
Note: For the solution it seems there is a typo and should read mapping=p9.aes(x='year', y='weight', color='species_id')) 

3. For the last bit about saving the plot, should the first part be my_plot=( p9. ...etc) ?
